CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to rpsrtsrs
+
+#### Table Of Contents
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
```

Comment:
There seem to be various tools (e.g. https://github.com/ekalinin/github-markdown-toc) that analyze a MarkDown file and rewrite it with a TOC, but the file would still have to be checked in with that change. GitHub [can generate a TOC for gh-pages](https://stackoverflow.com/a/9898502/674064) but [apparently not for the in-repo view of MarkDown files](https://github.com/isaacs/github/issues/215).

The document structure was <del>stolen from</del><ins>inspired by</ins> https://github.com/atom/atom/blob/master/CONTRIBUTING.md